api/envoy/config/filter/http/jwt_authn/v2alpha/config.proto

```diff
@@ -11,10 +11,19 @@ import "google/protobuf/empty.proto";
 import "google/protobuf/wrappers.proto";
 import "validate/validate.proto";
 
-// This message specifies how a JSON Web Token (JWT) can be verified. JWT format is defined
-// `here <https://tools.ietf.org/html/rfc7519>`_. Please see `OAuth2.0
-//  <https://tools.ietf.org/html/rfc6749>`_ and `OIDC1.0  <http://openid.net/connect>`_ for
-// the authentication flow.
+// Please see following for JWT authentication flow:
+//
+// * `JWT format <https://tools.ietf.org/html/rfc7519>`_
```

Comment:
nit: Can you update the link title matches the document linked to?
i.e. `JSON Web Token (JWT)`, `The OAuth 2.0 Authorization Framework`, `OpenID Connect`